src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +51,18 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User bob = authenticationService.register("bob@gmail.com", "qwerty");
+        User loggedBob = authenticationService.login("bob@gmail.com", "qwerty");
```

Comment:
Maybe you could add some System.out.println() to make sure methods are run.
Also methods throw exceptions, so it would be good to catch them -> use try and catch clause.
What would happen if you would try to add user with null password? Let's print it.